components/PicklistBase/PicklistBase.content.comp.cy.js

```diff
@@ -175,14 +206,18 @@ describe('Test the default PicklistBase content', () => {
     cy.get('@readySpy')
       .should('have.been.calledOnce')
       .then(() => {
-        cy.get('[data-cy="picklist-quantity-0"]').should('exist');
-        cy.get('[data-cy="picklist-quantity-0-0"]').should('exist');
-        cy.get('[data-cy="picklist-quantity-0-1"]').should('exist');
-
-        cy.get('[data-cy="picklist-quantity-1"]').should('exist');
-        cy.get('[data-cy="picklist-quantity-1-0"]').should('exist');
-        cy.get('[data-cy="picklist-quantity-1-1"]').should('exist');
-        cy.get('[data-cy="picklist-quantity-1-2"]').should('exist');
+        cy.get('select[data-cy="picklist-quantity-0"]')
+          .should('exist')
+          .and('have.value', '1');
+        cy.get('option[data-cy="picklist-quantity-0-0"]').should('exist');
+        cy.get('option[data-cy="picklist-quantity-0-1"]').should('exist');
```

Comment:
Since the quantities were changed above to 3 and 5,  this test should check for `picklist-quantity-0-2`.  Just checking for the first and last optios should be sufficient. Same comment applies a few lines below where you should check `...1-0` and `...1-4`